docs/flow/actions/string_similarity_action.md

```diff
@@ -0,0 +1,31 @@
+# String similarity
+
+This plugin checks similarity between two strings according to similarity algorithms.
+
+# Configuration
+
+Example:
+
+```json
+{
+   "first_string": "event@properties.some_value1",
+   "second_string": "event@properties.some_value2",
+   "searching_algorithm": "Levenshtein"
+}
+```
+
+# Input
+
+This plugin takes payload as input
+
+# Output
+
+Returns result of the searching algorithm
+
+Example:
+
+```json
+{
+  "similarity": "1"
```

Comment:
Describe what 1 means. What is the metric range. I assume 1 means the same (fully similar).